packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -40,8 +40,8 @@ duplicate-transaction = Hmm. Looks like an identical transaction was just sent.
 coupon-expired = It looks like that promo code has expired.
 card-error = Your transaction could not be processed. Please verify your credit card information and try again.
 
-fxa-signup-error = There was a problem creating your account.  Please try again later.
-fxa-newsletter-signup-error = There was a problem subscribing you to the newsletter.
+fxa-account-signup-error = A system error caused your ${ productName } sign-up to fail. Your payment method has not been charged. Please try again.
```

Comment:
Unsure if we need to leave comment to explain what $productName is to the localizers.  Couldn't hurt though.

Reply:
Whoops I missed this; I [asked Vijay](https://github.com/mozilla/fxa/pull/10138#issuecomment-900707371) to add a comment if possible in his WIP PR for the success screen.